configure.py

```diff
@@ -274,7 +274,7 @@ def LinkedFor(*versions):
             Object(LinkedFor("ce-j"), "emulator/mcardGCN.c"),
             Object(Linked, "emulator/codeGCN.c"),
             Object(Linked, "emulator/soundGCN.c"),
-            Object(LinkedFor("ce-j"), "emulator/frame.c", asm_processor=True),
+            Object(LinkedFor("ce-j"), "emulator/frame.c"),
```

Comment:
```suggestion
            Object(LinkedFor("mq-u", "mq-e", "ce-j", "ce-u", "ce-e"), "emulator/frame.c"),
```
I tried and this seems to match for those, also you might need to run `dol-apply`

Reply:
I think I got baited by my build folder, I'm recompiling just in case but I think it should match all versions but mq-j

Reply:
well, I'm confused, when I link for the other versions it acts the same as if it wasn't linked lol, but on objdiff it's matching, mq versions might require a dol-apply but that's it

Reply:
hm I built from clean and it worked